kv/key.go

```diff
@@ -355,8 +367,13 @@ func (ch *CommonHandle) String() string {
 }
 
 // MemUsage implements the Handle interface.
-func (ch *CommonHandle) MemUsage() int64 {
-	return int64(cap(ch.encoded)) + int64(cap(ch.colEndOffsets))*2
+func (ch *CommonHandle) MemUsage() uint64 {
+	return 48 + ch.ExtraMemSize()
```

Comment:
What 48 means?

Reply:
2 slice fields in CommonHandle. I'll add a comment.